.buildkite/full_pipeline_trigger.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+
+if [[ "$BUILDKITE_MESSAGE" == *"[QUICK CI]"* ]]; then
+  PIPELINE='pipeline.smoke.yml'
+elif [[ "$BUILDKITE_MESSAGE" == *"[FULL CI]"* ||
+  "$BUILDKITE_BRANCH" == "next" ||
+  "$BUILDKITE_BRANCH" == "master" ]]; then
```

Comment:
We'll also want full ci if it's a PR into `next` or `master`.  I think you can use `BUILDKITE_PULL_REQUEST_BASE_BRANCH` for that.